style.css

```diff
@@ -16,279 +16,524 @@ body {
   font-weight: 700;
 }
 
-@media (max-width: 480px) {
+.nav-flex {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 15px 20px;
+}
+
+nav button {
+  border: none;
+}
+
+nav ul {
+  display: none;
+}
+
+.headline {
+  background-image: url("./images/header-shapes\ mobile@2x.svg");
+}
+
+.headline .container {
+  margin: 0 20px;
+  height: 587.15px;
+  padding-top: 115px;
+  display: flex;
+  flex-direction: column;
+}
+
+.intro {
+  font-size: 28px;
+  color: #172b4d;
+  font-weight: 900;
+}
+
+.primary-text {
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.social {
+  text-transform: uppercase;
+  color: #7f8cff;
+  font-weight: 500;
+  padding: 10px 0;
+}
+
+.social ul {
+  width: 50%;
+  padding-top: 7px;
+  list-style: none;
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+}
+
+#works .container {
+  display: grid;
+  row-gap: 88px;
+  background-color: #fbfbfb;
+  padding: 114px 0;
+}
+
+#works .card {
+  height: 552px;
+  padding: 16px;
+
+  /* gap: 12px; */
+  border: 1px solid #dfe1e6;
+  background-color: white;
+  border-radius: 16px;
+  margin: auto 25px;
+}
+
+.card img {
+  width: 100%;
+}
+
+.texts h1 {
+  color: #172b4d;
+  font-size: 32px;
+}
+
+.below {
+  width: 80%;
+  color: #7a869a;
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+  font-size: 13px;
+  font-weight: 600;
+}
+
+.below .canopy {
+  text-transform: uppercase;
+  color: #344563;
+  font-weight: 600;
+}
+
+.below img {
+  max-height: 8px;
+  width: 8px;
+}
+
+.card .desc {
+  color: #344563;
+  margin: 10px 0;
+}
+
+.skills ul {
+  padding-top: 10px;
+  width: 75%;
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+  color: #6070ff;
+}
+
+.skills li {
+  background-color: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+}
+
+.card button,
+.btn-resume {
+  margin: 28px 0;
+  border: 1px solid #3965f2;
+  background-color: white;
+  padding: 8px;
+  display: flex;
+  text-align: center;
+  color: #3965f2;
+  letter-spacing: 0.03em;
+  font-weight: 500;
+  font-size: 17px;
+  border-radius: 8px;
+}
```

Comment:
You have done greatly at implementing a hover button interaction. Kindly implement all the button interactions to follow your chosen design template. 

Your design:
Enabled ![image](https://user-images.githubusercontent.com/20554583/193038052-e4d17782-0320-405c-b857-991ecc8e0324.png) hover ![image](https://user-images.githubusercontent.com/20554583/193038080-cb5ea1d2-f916-4801-bccb-0f494a88143f.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/193038200-f9b8f0ff-f8a7-493f-b797-cc364caddd87.png)